scripts/Jenkinsfile

```diff
@@ -36,9 +36,11 @@ pipeline
                     sh '''
                     . preflight_qeff/bin/activate
                     export TOKENIZERS_PARALLELISM=false
-                    pytest tests --ignore tests/cloud --junitxml=tests/tests_log1.xml
-                    pytest tests/cloud --junitxml=tests/tests_log2.xml
-                    junitparser merge tests/tests_log1.xml tests/tests_log2.xml tests/tests_log.xml
+                    pytest tests -m "not cli and not on_qaic" -n auto --junitxml=tests/tests_log1.xml &
+                    pytest tests -m "not cli and on_qaic" -n 4 --junitxml=tests/tests_log2.xml &
+                    wait
+                    pytest tests -m cli --junitxml=tests/tests_log3.xml
```

Comment:
this can also be run with -n 4

Reply:
We can run other tests that don't need qaic with value more than `-n 4`. So using the marker to filter, we're able to run much faster.

Reply:
I understood that part, but we can still run the tests on ran by line 42 with 4 threads